### PR TITLE
[#114] Add robot test tags config for "Deploy legion job"

### PR DESCRIPTION
**Pull request by ablatov**
Add TestTags parameter for customising tests amount by tags.

### CODE REVIEW

**Comment by MaksymBilenko** [comment]
ok to test

**Comment by MaksymBilenko** [comment]
Test this please

**Comment by MaksymBilenko** [comment]
Test this please

File: deploy/legionPipeline.groovy [hunk 1]
```diff
@@ -90,9 +90,10 @@ def createjenkinsJobs(String commitID) {
     """
 }
 
-def runRobotTests() {
+def runRobotTests(String tags) {
```
Comment (kirillmakhonin) [inline deploy/legionPipeline.groovy]: Maybe it will be better to add default value for compatibility with other installations where it is used?

**Comment by ablatov** [comment]
@kirillmakhonin now tags = "" by default

File: deploy/legionPipeline.groovy (hunk 1 above)
Comment (ablatov) [reply deploy/legionPipeline.groovy]: Done

**Pull request merged**